package.json

```diff
@@ -15,7 +15,8 @@
         "unit": "turbo run unit",
         "e2e": "turbo run e2e",
         "changeset": "changeset",
-        "release": "turbo run release && changeset publish"
+        "release": "turbo run release && changeset publish",
+        "download:env": "op read op://gitbook-x-dev/gitbook-open/.env.local > .env.local"
```

Comment:
```suggestion
        "download:env": "op read op://gitbook-x-dev/gitbook-open/.env.local >> .env.local"
```

Reply:
It means it could only be run once 🤔  I don't know if it's better but ok